src/attributes/prop.js

```diff
@@ -79,6 +79,12 @@ jQuery.extend( {
 	}
 } );
 
+// Support: IE<=11
```

Comment:
Edge is ok. Updated comment, but the syntax says there's no space after the `<=`, so it's `IE <=11 only`.